### ISSUE TITLE
HTTP Status 404 – Not Found

### ISSUE THREAD

**Issue by MaxDragonheart** [MaxDragonheart/docker-geoserver#41]
When I go to `127.0.0.1:8300/geoserver` I see this error:
```
HTTP Status 404 – Not Found
Type Status Report

Message The requested resource [/geoserver] is not available

Description The origin server did not find a current representation for the target resource or is not willing to disclose that one exists.

Apache Tomcat/9.0.84
```

**Comment by MaxDragonheart** [comment]
I don't understand why I can't use Geoserver versions above the 2.20.4

**Comment by MaxDragonheart** [comment]
It's possible to use Geoserver from 2.15.5(tested) to 2.20.4. Versions above 2.20.4 return 404

**Comment by MaxDragonheart** [comment]
> When I go to `127.0.0.1:8300/geoserver` I see this error:

Now is `127.0.0.1:8080/geoserver`

**Comment by MaxDragonheart** [comment]
Solved with #47
